apps/showcase/src/components/utilities/otter-picker/otter-icon-path.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { OTTER_ICONS } from './otter-icons';
+
+@Pipe({
+  name: 'otterIconPath',
+  standalone: true
+})
+export class OtterIconPathPipe implements PipeTransform {
+  private readonly BASE_URL = location.href.split('/#', 1)[0];
+  private readonly ICON_MATCHER = new RegExp(OTTER_ICONS.join('|'));
+
+  public transform(value: string) {
+    const iconPath = this.ICON_MATCHER.test(value) ? value : OTTER_ICONS[0];
```

Comment:
you could control completely the value and try to look for the value in the list of supported icons ?